packages/consts/src/knight-hacks.ts

```diff
@@ -42,9 +42,9 @@ export const EVENT_POINTS: Record<EventTag, number> = {
 
 export const KNIGHTHACKS_MEMBERSHIP_PRICE = 2500;
 
-export const DISCORD_ADMIN_ROLE_ID = "1319413082258411652";
+export const DISCORD_ADMIN_ROLE_ID = "1321955700540309645";
 
-export const KNIGHTHACKS_GUILD_ID = "486628710443778071";
+export const KNIGHTHACKS_GUILD_ID = "1151877367434850364";
```

Comment:
If these were switched for testing then they should be switched back

Reply:
whoops, big fumble ty